AlphaFold predictions failing

Some AlphaFold predictions are failing right now as AlphaFold API returns more entries than expected.
Example log:

```
2025-11-08T14:22:39 [INFO] : Preparing structure ...
2025-11-08T14:22:39 [DEBUG] : Starting new HTTPS connection (1): alphafold.ebi.ac.uk:443
2025-11-08T14:22:43 [DEBUG] : https://alphafold.ebi.ac.uk:443 "GET /api/prediction/Q5VSL9 HTTP/1.1" 200 None
2025-11-08T14:22:43 [ERROR] : Execution failed.
Traceback (most recent call last):
  File "/opt/executor-p2rank/run_p2rank_task.py", line 127, in _execute_directory_task
    result = execute(execution)
             ^^^^^^^^^^^^^^^^^^
  File "/opt/executor-p2rank/executor.py", line 26, in execute
    structure = _prepare_structure(configuration)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/executor-p2rank/executor.py", line 69, in _prepare_structure
    raw_structure_file = _prepare_raw_structure_file(configuration, metadata)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/executor-p2rank/executor.py", line 101, in _prepare_raw_structure_file
    _download_from_alpha_fold(
  File "/opt/executor-p2rank/executor.py", line 135, in _download_from_alpha_fold
    assert len(entry_content) == 1, \
           ^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: One entry expected for AlphaFold, found 4
```

The fix is to replace the assert with something like this:

```python
 if len(entry_content) != 1:
        logger.warning(f"One entry expected for AlphaFold, found {len(entry_content)}! Using the first entry.")
```

Will do it as soon as possible.

--- a/executor-p2rank/executor.py
+++ b/executor-p2rank/executor.py
@@ -132,8 +132,8 @@ def _download_from_alpha_fold(
     metadata["alpha-fold"] = entry_content
     if len(entry_content) == 0:
         raise Exception(f"No Alphafold entry found for: {code}")
-    assert len(entry_content) == 1, \
-        f"One entry expected for AlphaFold, found {len(entry_content)}"
+    if len(entry_content) != 1:
+        logger.warning(f"One entry expected for AlphaFold, found {len(entry_content)}! Using the first entry.")
     cif_url = entry_content[0]["cifUrl"]
     _download(cif_url, destination)
 

